commit 8eb94d82ee24d9d0e4f75b2d8e194f651dab444b (refs/changes/97/2997/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-11-24T16:38:36-08:00 (1 year, 10 months ago)
    
    VOLAPI-96 sdcadm post-setup volapi errors when default channel is not "dev"

diff --git a/lib/post-setup/volapi.js b/lib/post-setup/volapi.js
index 504c996..39aab97 100644
--- a/lib/post-setup/volapi.js
+++ b/lib/post-setup/volapi.js
@@ -250,8 +250,7 @@ function do_volapi(subcmd, opts, args, cb) {
             proc.execute({
                 sdcadm: self.sdcadm,
                 log: self.log,
-                progress: self.progress,
-                source: 'https://updates.joyent.com'
+                progress: self.progress
             }, next);
         },
 
